hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsHttpOperation.java

```diff
@@ -74,6 +76,7 @@
   // metrics
   private int bytesSent;
   private long bytesReceived;
+  private long bytesDiscarded;
```

Comment:
nit, javadocs for the new attribute: try to do better than your predecessors....

Reply:
Have added the javadocs. Will keep a PR checklist point on this. Thanks.